docs/topics/SSHAgent.adoc

```diff
@@ -0,0 +1,125 @@
+
+To use Pageant with KeePassXC, simply start it and it will minimize into the system tray and is ready to use. PuTTY and compatible tools will use Pageant automatically.
+
+=== Setting up SSH Agent integration
+By default the SSH Agent integration plugin is disabled.
+To enable integration, follow the steps below to access the settings:
+
+ 1. Select _Tools > Settings_ from the menu
+ 2. Select _SSH Agent_ category on the left sidebar
+
+.SSH Agent Application Settings Page
+image::sshagent_application_settings.png[]
+
+On the settings page you can enable the integration by checking _Enable SSH Agent integration_.
+When the integration is enabled coming back to the settings page also shows if connection to the agent is working.
+
+On Windows you have the option to select between _Pageant_ and _OpenSSH for Windows_ and on other platforms the settings page shows the current value of _SSH_AUTH_SOCK_ environment variable which is used to connect to the running agent and an option to manually override the automatically detected path.
+
+If the value of _SSH_AUTH_SOCK_ is empty it means the agent is not properly configured and KeePassXC will be unable to connect to it unless you provide a static override path to the socket.
+
+=== Generating a key to use with KeePassXC
+KeePassXC only supports keys in the _OpenSSH_ format. On Windows, _PuTTYgen_ saves keys in its own format by default and you will need to convert them to OpenSSH format before being used. In this guide we are going to generate a standard RSA key in the default size.
+
+==== Generating a key on Linux or MacOS with _ssh-keygen_
+
+Open a terminal window and type the following command to generate a key:
+
+    $ ssh-keygen -o -f keepassxc -C johndoe@example
+    Generating public/private rsa key pair.
+    Enter passphrase (empty for no passphrase): 
+    Enter same passphrase again: 
+    Your identification has been saved in keepassxc
+    Your public key has been saved in keepassxc.pub
+    The key fingerprint is:
+    SHA256:pN+o5AqUmijYBDUrFV/caMus9oIR61+MiWLa8fcsVYI johndoe@example
+    The key's randomart image is:
+    +---[RSA 3072]----+
+    |  =. ..o         |
+    | o + .+ .        |
+    |o . .+ o.        |
+    | o..  Eo. .      |
+    |  +o .. So       |
+    |o*o.o+ ..o       |
+    |Bo=+o.+.o .      |
+    |+oo+.++o         |
+    |. ..++ooo        |
+    +----[SHA256]-----+
+
+
+Now we can see two files wre generated:
+
+    $ ls -l keepassxc*
+    -rw------- 1 user group 2.6K Apr  5 07:36 keepassxc
+    -rw-r--r-- 1 user group  569 Apr  5 07:36 keepassxc.pub
+
+With KeePassXC you only need the first file listed.
+
+==== Generating a key on Windows with PuTTYgen
+Please read the manual on how to use PuTTYgen for details on generate a key: https://www.ssh.com/ssh/putty/putty-manuals/0.68/Chapter8.html#pubkey-puttygen. Once generated, you must save the key in OpenSSH format, follow the image below.
+
+.Generating Key with PuttyGen

```

Comment:
Capitalization of PuTTYgen is wrong here and later. It should be `PuTTYgen` everywhere.